gcc/rust/privacy/rust-visibility-resolver.cc

```diff
@@ -45,6 +45,17 @@ VisibilityResolver::go (HIR::Crate &crate)
     }
 }
 
+// FIXME: At this point in the pipeline, we should not be dealing with
+// `AST::SimplePath`s anymore! We need to be dealing with their "resolved
+// counterpart", so probably a NodeId/HirId/DefId.
+
+// static bool
+// resolve_module_path (std::vector<HIR::Module> &module_stack,
+// 		     const AST::SimplePath &restriction, DefId &id)
+// {
+//   return false;
```

Comment:
yep there will be an HIR simple path which we might desugar but this should insert into the name/type namespace what the resolved NodeId is then we just use this in the rest of the pipeline.

Reply:
Not sure this change needs to be in this patch though.

Reply:
Yeah this part is still very early and I think it should be changed so that the `HIR::Visibility` does not keep a `SimplePath` but simply its associated `DefId` as you said. Do you want me to remove the comment? It was simply for later on to keep track of stuff I need to do in the Privacy visibility resolver